docker-compose.yml

```diff
@@ -1,195 +1,81 @@
-        environment:
-            <<: *php-environment
-        depends_on:
-            - consul
-            - postgres
-            - rabbitmq
-        command: bin/console messenger:consume --sleep=10 --memory-limit=128M --time-limit=3600 -vv import channel export core event segment
-        healthcheck:
-            disable: true
-        restart: on-failure
+
 services:
+
     postgres:
-        build:
-            context: "./"
-            target: postgres
-            <<: *ergonode-cache-from
-        image: ${CONTAINER_REGISTRY_BASE:-ergonode}/postgres:${IMAGE_TAG:-latest}
+        image: "postgres:10"
         volumes:
             - "./data/.postgres:/var/lib/postgresql/data:delegated"
+            - "./config/postgres/init.sql:/docker-entrypoint-initdb.d/init.sql"
         environment:
-            POSTGRES_USER: ${POSTGRES_USER:-postgres}
-            POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-123}
-            POSTGRES_DB: ${POSTGRES_DB:-postgres}
-            APP_USER: ${APP_USER:-ergonode}
-            APP_USER_PASSWORD: ${APP_USER_PASSWORD:-123}
-            APP_DB: ${APP_DB:-ergonode}
-            APP_TEST_DB: ${APP_TEST_DB:-ergonode_test}
+            POSTGRES_USER: "postgres"
+            POSTGRES_PASSWORD: "123"
+            POSTGRES_DB: "postgres"
         ports:
-            - "${EXPOSED_POSTGRES_PORT:-5432}:5432"
+            - "5432"
+
     php:
         build:
             context: "./"
-            target: php
-            <<: *ergonode-cache-from
-        working_dir: /srv/app
-        image: ${CONTAINER_REGISTRY_BASE:-ergonode}/php:${IMAGE_TAG:-latest}
+            dockerfile: "./config/php/Dockerfile"
+        working_dir: "/srv/app"
+        entrypoint: "/entrypoint.sh"
         volumes:
-            - ./backend:/srv/app:delegated
+            - "./backend:/srv/app:delegated"
+            - "./config/php/cron:/etc/cron.d:delegated"
+            - "./config/php/conf.d/php-ini-override.ini:/usr/local/etc/php/conf.d/php-ini-override.ini"
+            - "./config/php/supervisor/supervisord.conf:/etc/supervisor/supervisord.conf"
+            - "./config/php/supervisor/conf.d:/etc/supervisor/conf.d:delegated"
+            - "./config/php/docker-entrypoint.sh:/entrypoint.sh:delegated"
         environment:
-            <<: *php-environment
+            XDEBUG_CONFIG: ${XDEBUG_CONFIG}
+            PHP_IDE_CONFIG: ${PHP_IDE_CONFIG}
+            APP_URL: "http://nginx"
+        links:
+            - "postgres"
         depends_on:
-            - postgres
-            - mailhog
-    crond:
-        build:
-            context: "./"
-            target: php
-            <<: *ergonode-cache-from
-        working_dir: /srv/app
-        image: ${CONTAINER_REGISTRY_BASE:-ergonode}/php:${IMAGE_TAG:-latest}
-        volumes:
-            - ./backend:/srv/app:delegated
-        environment:
-            <<: *php-environment
-        command: ["crond"]
+            - "postgres"
+            - "mailhog"
+            - "rabbitmq"
```

Comment:
You can have synchronous local configuration and don't want to raise extra containers

Reply:
Sure, no problem